### PR TITLE
full-text docs search

### PR DESCRIPTION
**Pull request by zramsay**
Woot Woot https://github.com/tendermint/devops/issues/117

it's only a search API so it's not a secret (approved by Jessy). Fixed the "Last Updated" feature which uses git and refers to each individual file

* [ ] Updated all relevant documentation in docs
* [ ] Updated all code comments where relevant
* [ ] Wrote tests
* [ ] Updated CHANGELOG_PENDING.md

### CODE REVIEW

File: docs/.vuepress/config.js [hunk 1]
```diff
@@ -8,7 +8,12 @@ module.exports = {
     lineNumbers: true
   },
   themeConfig: {
-    lastUpdated: "Last Updated",
+    lastUpdated: true,
+    algolia: {
+      apiKey: '59f0e2deb984aa9cdf2b3a5fd24ac501',
```
Comment (melekes) [inline docs/.vuepress/config.js]: this is search-only API key (https://www.algolia.com/doc/guides/security/api-keys/?language=php#search-only-api-key), right? just want to make sure

Comment (zramsay) [reply docs/.vuepress/config.js]: yes

**Comment by zramsay** [comment]
linking https://github.com/algolia/docsearch-configs/pull/668 which we can make additional PRs to update the config

**Pull request merged**